src/org/rascalmpl/uri/URIResolverRegistry.java

```diff
@@ -402,9 +402,12 @@ private ISourceLocation physicalLocation(ISourceLocation loc) throws IOException
 			ILogicalSourceLocationResolver resolver = map.get(auth);
 			loc = resolveAndFixOffsets(loc, resolver, map.values());
 		}
-		var fallBack = fallbackLogicalResolver;
-		if (fallBack != null) {
-			return resolveAndFixOffsets(loc == null ? original : loc, fallBack, Collections.emptyList());
+		if (loc == null) {
```

Comment:
this is not the right place for this check, it changes the logic of the case where there is no `fallbackLogicalResolver` defined.